### PR TITLE
Start balsamic with Trailblazer

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR makes trailblazer a bit more general and tries to encourage starting balsamic through it

**How to test**:
1. install trailblazer on stage of the hasta machine: `bash servers/resources/hasta.scilifelab.se/update-trailblazer-stage.sh mvp-balsamic`
1. install cg on stage of the hasta machine: `bash servers/resources/hasta.scilifelab.se/update-cg-stage.sh mvp-balsamic`
1. activate stage: `us`
1. run following command: `trailblazer start --script balsamic-bin-path -c BALSAMIC-CONFIG CASE-ID`

**Expected outcome**:
`trailblazer` should try to start balsamic with the binary and the config provided 
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @ingkebil 
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This is a minor **version bump** because it adds functionality in a backwards compatible manner

### CODE REVIEW

File: trailblazer/cli/core.py [hunk 1]
```diff
@@ -69,31 +69,38 @@ def log_cmd(context, sampleinfo, sacct, quiet, config):
 
 
 @base.command()
-@click.option('-c', '--mip-config', type=click.Path(exists=True), help='MIP config')
+@click.option('-c', '--mip-config', 'pipeline_config', type=click.Path(exists=True),
```
Comment (ingkebil) [inline trailblazer/cli/core.py]: Shouldn't the option be named something different than mip-config?

File: trailblazer/cli/core.py [hunk 2]
```diff
@@ -69,31 +69,38 @@ def log_cmd(context, sampleinfo, sacct, quiet, config):
...
+              help='Pipeline config')
 @click.option('-e', '--email', help='email for logging user')
 @click.option('-p', '--priority', type=click.Choice(['low', 'normal', 'high']), default='normal')
 @click.option('-d', '--dryrun', is_flag=True, help='only generate SBATCH scripts')
-@click.option('--command', is_flag=True, help='only show the MIP command')
-@click.option('-sw', '--start-with', help='start the pipeline beginning with program,\
-                                           see format for program in mip.pl')
+@click.option('--pipeline', help='sub-pipeline to run')
+@click.option('--script', help='binary path')
+@click.option('--command', is_flag=True, help='only show the pipeline command')
```
Comment (ingkebil) [inline trailblazer/cli/core.py]: I now this ain't new. `command` is so meaningless. Can we rename it to `show-command` or something more descriptive?

File: trailblazer/cli/core.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply trailblazer/cli/core.py]: My intention was to change stuff without breaking compatibility and this seemed to have the potential to break something

File: trailblazer/cli/core.py (hunk 2 above)
Comment (patrikgrenfeldt) [reply trailblazer/cli/core.py]: same here, I have as a goal in this PR not to break the application :)

File: trailblazer/cli/core.py (hunk 1 above)
Comment (ingkebil) [reply trailblazer/cli/core.py]: Alright!

File: trailblazer/cli/core.py (hunk 2 above)
Comment (ingkebil) [reply trailblazer/cli/core.py]: Alright!

**Comment by patrikgrenfeldt** [comment]
I think this PR has been obsoleted, closing

**Pull request closed without merging**